tests/fixtures/restrictions.ts

```diff
@@ -1,118 +1,161 @@
-    },
-  ],
-  MatchInputOutputMetadataValue: [
-    {
-      inputIndex: 0,
-      inputMetadataKey: 'SomeMetadataKey',
-      outputIndex: 0,
-      outputMetadataKey: 'SomeMetadataKey',
-    },
-  ],
-  FixedNumberOfInputs: [
-    {
-      numInputs: 0,
-    },
-  ],
-  FixedNumberOfOutputs: [
-    {
-      numOutputs: 0,
-    },
-  ],
-  FixedInputMetadataValue: [
-    {
-      index: 0,
-      metadataKey: 'SomeMetadataKey',
-      metadataValue: {
-        Literal: 'a',
+export const validAllRestrictions: Process.Program = [
+  { restriction: { None: {} } },
+  { op: 'or' },
+  { restriction: { SenderOwnsAllInputs: {} }},
+  { op: 'and' },
+  { restriction: {
+    SenderHasInputRole: 
+      {
+        index: 0,
+        roleKey: 'Supplier',
       },
-    },
-  ],
-  FixedOutputMetadataValue: [
-    {
-      index: 0,
-      metadataKey: 'SomeMetadataKey',
-      metadataValue: {
-        Literal: 'a',
+    
+  }},
+  { op: 'and' },
+  { restriction: {
+    SenderHasOutputRole: 
+      {
+        index: 0,
+        roleKey: 'Supplier',
       },
-    },
-  ],
-  FixedOutputMetadataValueType: [
-    {
-      index: 0,
-      metadataKey: 'SomeMetadataKey',
-      metadataValueType: 'Literal',
-    },
-  ],
-})
+    
+  }},
+  { op: 'and' },
+  { restriction: {
+    OutputHasRole: 
+      {
+        index: 0,
+        roleKey: 'Supplier',
+      },
+    
+  }},
+  { op: 'and' },
+  { restriction: {
+    MatchInputOutputRole: 
+      {
+        inputIndex: 0,
+        inputRoleKey: 'Supplier',
+        outputIndex: 0,
+        outputRoleKey: 'Supplier',
+      },
+    
+  }},
+  { op: 'and' },
+  { restriction: {
+    MatchInputOutputMetadataValue: 
+      {
+        inputIndex: 0,
+        inputMetadataKey: 'SomeMetadataKey',
+        outputIndex: 0,
+        outputMetadataKey: 'SomeMetadataKey',
+      },
+    
+  }},
+  { op: 'and' },
+  { restriction: {
+    FixedNumberOfInputs: 
+      {
+        numInputs: 0,
+      },
+    
+  },},
+  { op: 'and' },
+  { restriction: {
+    FixedNumberOfOutputs: 
+      {
+        numOutputs: 0,
+      },
+    
+  },},
+  { op: 'and' },
+  { restriction: {
+    FixedInputMetadataValue: 
+      {
+        index: 0,
+        metadataKey: 'SomeMetadataKey',
+        metadataValue: {
+          Literal: 'a',
+        },
+      },
+    
+  },},
+  { op: 'and' },
+  { restriction: {
+    FixedOutputMetadataValue: 
+      {
+        index: 0,
+        metadataKey: 'SomeMetadataKey',
+        metadataValue: {
+          Literal: 'a',
+        },
+      },
+    
+  },},
+  { op: 'and' },
+  { restriction: {
+    FixedOutputMetadataValueType: 
+      {
+        index: 0,
+        metadataKey: 'SomeMetadataKey',
+        metadataValueType: 'Literal',
+      },
+    
+  },},
+]
 
-export const validMultipleOfSameRestrictions = JSON.stringify({
-  FixedInputMetadataValue: [
+export const validMultipleOfSameRestrictions: Process.Program = [ 
+  { restrintion: { FixedInputMetadataValue: 
```

Comment:
typo on `restriction`, although looks like `validMultipleOfSameRestrictions` isn't used anymore so could delete whole thing

Reply:
hehe.. this was a test. I guess I was just lucky that this test is no longer relevant. Thanks, will remove.

Reply:
Updated.